.github/workflows/netlify.yml

```diff
@@ -29,6 +29,7 @@ jobs:
           crawler-user-id: ${{ secrets.CRAWLER_USER_ID }}
           crawler-api-key: ${{ secrets.CRAWLER_API_KEY }}
           crawler-api-base-url: 'https://crawler-dev.algolia.com/api/1/'
+          github-token: ${{ secrets.GIT_HUB_TOKEN }}
```

Comment:
mmh I believe github actions comes with a github token, can you check other implementation?

Reply:
Apparently, this issue has been already raised ;) : https://github.com/peaceiris/actions-gh-pages/issues/309

Reply:
gonna set the default as in the example